cassandra-5.0.yaml

```diff
@@ -102,6 +103,45 @@ subpackages:
             virtual-pkg-name: cassandra-compat
             real-pkg-name: ${{subpkg.name}}
 
+  # This subpackage provides compatibility with upstream docker-library/cassandra entrypoint behavior.
+  # It fetches the official entrypoint script from GitHub and verifies its integrity using SHA256.
+  # The entrypoint script processes environment variables (CASSANDRA_SEEDS, CASSANDRA_CLUSTER_NAME,
+  # CASSANDRA_DC, CASSANDRA_RACK, etc.) and writes them to configuration files.
+  #
+  # This is necessary for sustaining because upstream may update the entrypoint script independently
+  # of Cassandra releases. By tracking the SHA256 hash, we can detect when the script changes and
+  # decide whether to update our vendored copy or investigate breaking changes.
+  #
+  # If the hash verification fails, the pipeline will exit 1 to alert that the upstream
+  # entrypoint has changed and requires review before accepting the update.
+  #
+  # Reference: https://github.com/chainguard-dev/customer-issues/issues/2563
+  - name: ${{package.name}}-entrypoint-compat
+    description: Entrypoint script for Cassandra environment variable configuration
+    dependencies:
+      runtime:
+        - bash
+        - busybox
```

Comment:
Do we need `busybox`? It's usually recommended to replace this with lighter component packages like `sed`, `coreutils` to reduce attack surface in the package/image.